libs/clients/national-registry/v3/src/lib/apiConfig.ts

```diff
@@ -0,0 +1,38 @@
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+import {
+  ConfigType,
+  LazyDuringDevScope,
+  XRoadConfig,
+} from '@island.is/nest/config'
+
+import { Configuration } from '../../gen/fetch'
+import { NationalRegistryV3ClientConfig } from './nationalRegistryV3.config'
+
+export const ApiConfig = {
+  provide: 'NationalRegistryV3ClientProviderConfiguration',
+  scope: LazyDuringDevScope,
+  useFactory: (
+    xroadConfig: ConfigType<typeof XRoadConfig>,
+    config: ConfigType<typeof NationalRegistryV3ClientConfig>,
+  ) =>
+    new Configuration({
+      fetchApi: createEnhancedFetch({
```

Comment:
In the current national registry client we configure our own redis caching to speed up lookups and provide `stale-if-error` handling when there are problems with the service.

I wonder if we can trust this new service to be more reliable and faster?

Reply:
reimplement the caching perhaps?